rust/src/sdp/parser.rs

```diff
@@ -40,13 +40,13 @@ use std::str::FromStr;
 #[derive(Debug)]
 pub struct SdpMessage {
     pub version: u32,
-    pub origin: OriginField,
+    pub origin: String,
```

Comment:
Why a String and not a Vec<u8> ?

Reply:
Why should I use a vec?

Reply:
Ok for me as all `String` are generated by `format!` thanks @glongo